frontend/js/src/playlists/Playlist.tsx

```diff
@@ -445,12 +450,25 @@ export default function PlaylistPage() {
           {playlist.annotation && (
             <div className="wikipedia-extract">
               <div
-                className="content"
+                className={`content ${showMore ? "playlist-description" : ""}`}
```

Comment:
With my comments above about reusing the existing class, this should be something like 
```suggestion
                className={`description ${showMore ? "expanded" : ""}`}
```